content/en/post/2022-05-19-nurturing-ct-log-growth.md

```diff
@@ -3,10 +3,10 @@ author: J.C. Jones
 date: 2022-05-19T00:00:00Z
```

Comment:
```suggestion
date: 2022-06-13T00:00:00Z
```

Reply:
Pretty sure we don't want to do this, as it's not the last modified time, it's the publication time.